src/middleware/fixImageUrlsMiddleware.js

```diff
@@ -7,7 +7,7 @@ const legacyUrls = [
 
 export const fixUrl = (url) => {
   legacyUrls.forEach((legacyUrl) => {
-    url = url.replace(legacyUrl, '')
+    url = url.replace(legacyUrl, '/api')
```

Comment:
@appinteractive probably not the best way to fix this. The backend *knows* frontend implementation details. It should be `''` not `'/api'` and the frontend image tags in the template needs to be changed, right?

Reply:
Yeah we should change that in the Frontend to prefix them with /api

Reply:
@robert?

Reply:
Uhm, we can merge that and fix it later? It requires a change in the fronted

Reply:
I just wanted to make we are on the same page here